plugins/janus_sip.c

```diff
@@ -749,10 +749,20 @@ static GHashTable *callids;
 static janus_mutex sessions_mutex = JANUS_MUTEX_INITIALIZER;
 
 static void janus_sip_srtp_cleanup(janus_sip_session *session);
+static void janus_sip_media_reset(janus_sip_session *session);
 
+static void update_session_status(janus_sip_session *session, janus_sip_call_status new_status) {
+	if (session->status != new_status) {
+		JANUS_LOG(LOG_VERB, "[%s], session status change [%s]-->[%s]\n", session->account.username == NULL ? "null" : session->account.username, janus_sip_call_status_string(session->status), janus_sip_call_status_string(new_status));
+		session->status = new_status;
+	}
+}
 
-
-static void update_session_status(janus_sip_session *session, janus_sip_call_status new_status);
+static int janus_sip_call_is_established(janus_sip_session *session) {
```

Comment:
I think this should be boolean, not int.

Reply:
Have changed. Thanks.